client/state/README.md

```diff
@@ -17,6 +17,26 @@ const store = createReduxStore();
 
 All the application information and data in Calypso should go through this data flow. When you are creating a new module with new data requirements, you should add them to this global store.
 
+You can add a new reducer to the store at runtime easily by wrapping your main component with the `withReducer` Higher Order Component:
+
+```js
+import { withReducer } from 'state/with-reducer';
+
+//...your main React component here...
+
+export default withReducer( 'myModule', myReducerFunction )( MyMainReactComponent );
+```
+
+Alternatively, you can use a more manual approach by adding and removing the reducer yourself. Just make sure you remove the reducer when your code becomes inactive within the system.
+
+```js
+// Call this when your code becomes active.
+store.addReducer( 'myModule', myReducerFunction );
+
+// Call this when your code becomes inactive.
+store.removeReducer( 'myModule' );
```

Comment:
`replaceReducer` is exposed as part of the store API and might be necessary for store enhancers, but shouldn't make it past a code review if someone tries to call it inappropriately. on the other hand, I don't actually have a problem hiding it altogether from everything past the core Redux subsystem.

the difference here I believe is the fact that we're putting in code in a guide on using `addReducer()` and `removeReducer()` and those things seem straightforward and _blessed_ by their inclusion. if we hide those by default and make people make active changes to use them we might head off inappropriate uses.

just asking the question: is this necessary? is there _any_ reasonable case we can imagine needing it? could we get away with keeping it out?